stretch_core/stretch_core/stretch_driver.py

```diff
@@ -5,6 +5,7 @@
 import numpy as np
 import threading
 from .rwlock import RWLock
+from typing import List
```

Comment:
`typing` is unnecessary with Python 3.9 and onward (22.04 uses 3.10.12), use `list` directly

https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc